mautic/mautic
# Feature sms web detail view

Please answer the following questions. 

| Q  | A
| --- | ---
| Bug fix? | N
| New feature? | Y
| Related user documentation PR URL | N/A
| Related developer documentation PR URL | N/A
| Issues addressed (#s or URLs) | N/A
| BC breaks? | N
| Deprecations? | N

#### Description:
This very simply adds detail view for Text Messages and Web Notifications.

#### Steps to test this PR:
1. Have a Web Notification and Text Message with successful sends.
2. In the list view for the items, click the item title in the table to load the detail view.
3. Ensure the "Clicks" and "Contacts" tabs are properly populated.
